src/stores/connectors/secrets/restore.go

```diff
@@ -9,27 +9,27 @@ import (
 	"github.com/consensys/quorum-key-manager/src/stores/database"
 )
 
-func (c Connector) Restore(ctx context.Context, id, version string) error {
-	logger := c.logger.With("id", id, "version", version)
+func (c Connector) Restore(ctx context.Context, id string) error {
+	logger := c.logger.With("id", id)
 	logger.Debug("restoring secret")
 
 	err := c.authorizator.CheckPermission(&types.Operation{Action: types.ActionDelete, Resource: types.ResourceSecret})
 	if err != nil {
 		return err
 	}
 
-	secret, err := c.db.GetDeleted(ctx, id, version)
+	secret, err := c.db.GetDeleted(ctx, id, "")
 	if err != nil {
 		return err
 	}
 
 	err = c.db.RunInTransaction(ctx, func(dbtx database.Secrets) error {
-		err = dbtx.Restore(ctx, secret.ID, version)
+		err = dbtx.Restore(ctx, secret.ID, "")
```

Comment:
I guess that we can also remove the `version` from the `Restore` interface?

Reply:
I was considering too since there is not usage of no empty version but I wanted to keep the implementation. But I believe you are right, better to reduce the code to maintain than keep it for JIC reasons